ui/appui-abstract/src/appui-abstract/properties/Record.ts

```diff
@@ -25,6 +25,19 @@ export interface LinkElementsInfo {
   matcher?: (displayValue: string) => Array<{ start: number, end: number }>;
 }
 
+/** JSON representation of a [[PropertyRecord]].
+ * @beta
+ */
+export interface PropertyRecordJSON {
```

Comment:
appui-abstract follows core with its interface which ends with "props", so this would be called `PropertyRecordProps`, `*JSON` is more in presentation where they have `React` included which have a specific meaning for the "props" term. (appui-abstract is expected to remain framework agnostic, so it should follow core naming conventions)